readme.md

```diff
@@ -8,7 +8,7 @@ From the root directory of your [Pico](http://pico.dev7studios.com/) installatio
 
 Once you’ve downloaded yourself a copy of the theme, open up the Pico configuration file (`./config.php`) and add the following lines:
 
-	$config['theme']          = 'flatui-blog';
+	$config['theme']          = 'pico-flatui-blog';
```

Comment:
not sure this change is correct unless you change the `git clone` command above too...